File: example_scroller/src/main/java/com/simple/scroller_demo/MainActivity.java
package com.simple.scroller_demo;

import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;

public class MainActivity extends AppCompatActivity {

    private ScrollerView mScrollerView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        mScrollerView = findViewById(R.id.scrollerView);
    }

    public void onClick(View view){
        mScrollerView.smoothScroll();
    }
}

File: example_custom_views/src/main/java/me/simple/example_custom_views/MainActivity.kt
package me.simple.example_custom_views

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import me.simple.example_custom_views.constructor.ConstructorActivity
import me.simple.example_custom_views.measure.MeasureActivity

class MainActivity : BaseActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun constructor(view: View) {
        navTo(ConstructorActivity::class.java)
    }

    fun measure(view: View) {
        navTo(MeasureActivity::class.java)
    }
}
